modules/system/console/MixCompile.php

```diff
@@ -81,7 +81,7 @@ public function handle(): int
 
         foreach ($registeredPackages as $name => $package) {
             $relativeMixJsPath = $package['mix'];
-            if (!$this->canCompilePackage($relativeMixJsPath)) {
+            if (!$this->canCompilePackage(base_path($relativeMixJsPath))) {
```

Comment:
Why is this required?

Reply:
relativeMixJsPath was not correctly run on npx thus base_path resolves this issue by providing fully qualified path of the project. Also, it makes the path explicit.